.github/workflows/build_and_test.yml

```diff
@@ -728,6 +729,68 @@ jobs:
         ./build/mvn $MAVEN_CLI_OPTS -DskipTests -Pyarn -Pmesos -Pkubernetes -Pvolcano -Phive -Phive-thriftserver -Phadoop-cloud -Djava.version=${JAVA_VERSION/-ea} install
         rm -rf ~/.m2/repository/org/apache/spark
 
+  connect-maven:
```

Comment:
friendly ping @HyukjinKwon @hvanhovell  I want to add  a independent maven testing GA task for connect modules due to there are always some Maven tests that fail, and we can only check them afterwards, see:
- https://issues.apache.org/jira/browse/SPARK-43646
- https://issues.apache.org/jira/browse/SPARK-43648
- https://issues.apache.org/jira/browse/SPARK-43647 

I found today, do you think it's ok?

Also cc @dongjoon-hyun